src/LaunchDarkly.ClientSdk/LdClient.cs

```diff
@@ -226,14 +234,43 @@ public sealed class LdClient : ILdClient
 
         void Start(TimeSpan maxWaitTime)
         {
+            if (maxWaitTime >= ExcessiveInitWaitTime)
+            {
+                _log.Warn(ExcessiveInitWaitTimeWarning, maxWaitTime, ExcessiveInitWaitTime);
+            }
+
             var success = AsyncUtils.WaitSafely(() => _connectionManager.Start(), maxWaitTime);
             if (!success)
             {
-                _log.Warn("Client did not successfully initialize within {0} milliseconds.",
-                    maxWaitTime.TotalMilliseconds);
+                _log.Warn(DidNotInitializeTimelyWarning, maxWaitTime.TotalMilliseconds);
             }
         }
 
+        /// <summary>
+        /// Starts the client and waits up to the wait time to initialize feature flags.  If offline,
```

Comment:
Should it mention that initialization will continue in the background, even if the timeout elapses?